components/Modules/VsBrSingleImageLinksModule.vue

```
@@ -107,7 +111,47 @@ const props = defineProps<{ module: object, theme: string }>();
 const module: any = props.module;
 const theme: string = props.theme;
 
+const media = ref(null);
+
+let negativeMargin = ref('200px');
```

ESLint

[error] 'negativeMargin' is never reassigned. Use 'const' instead.

(prefer-const)